db/migrate/20150930183738_migrate_content_hosts.rb

```diff
@@ -0,0 +1,213 @@
+    self.table_name = "katello_system_host_collections"
+  end
+
+  class Katello::ErrataAssociation < ActiveRecord::Base
+    self.table_name = "katello_system_errata"
+  end
+
+  class Katello::RepositoryAssociation < ActiveRecord::Base
+    self.table_name = "katello_system_repositories"
+  end
+
+  class Katello::System < ActiveRecord::Base
+    def backend_data
+      @data ||= ::Katello::Resources::Candlepin::Consumer.get(uuid)
+    end
+
+    def facts
+      backend_data[:facts]
+    end
+
+    self.table_name = "katello_systems"
+
+    has_many :system_activation_keys, :class_name => "Katello::SystemActivationKey", :dependent => :destroy
+    has_many :activation_keys,
+             :through => :system_activation_keys,
+             :after_add    => :add_activation_key,
+             :after_remove => :remove_activation_key
+    belongs_to :content_view, :inverse_of => :systems
+    belongs_to :environment, :class_name => "Katello::KTEnvironment", :inverse_of => :systems
+    has_many :errata_associations, :class_name => "Katello::ErrataAssociation", :dependent => :destroy
+    has_many :repository_associations, :class_name => "Katello::RepositoryAssociation", :dependent => :destroy
+    has_many :host_collections, :class_name => "Katello::HostCollectionAssociation", :dependent => :destroy
+    has_many :applicable_errata, :through => :system_errata, :class_name => "Katello::Erratum", :source => :erratum
+    has_many :system_errata, :class_name => "Katello::SystemErratum", :dependent => :destroy, :inverse_of => :system
+    has_many :bound_repositories, :through => :system_repositories, :class_name => "Katello::Repository", :source => :repository
+    has_many :system_repositories, :class_name => "Katello::SystemRepository", :dependent => :destroy, :inverse_of => :system
+
+  end
+
+  def logger
+    Rails.logger
+  end
+
+  def create_content_aspect(host, system)
+    logger.info("Creating content aspect for host #{host.name}.")
+    content_aspect = host.content_aspect = ::Katello::Host::ContentAspect.new(:content_view => system.content_view,
+                                                                              :lifecycle_environment => system.environment)
+    content_aspect.uuid = system.uuid
+    content_aspect.bound_repositories = system.bound_repositories
+    content_aspect.applicable_errata = system.applicable_errata
+    content_aspect.save!
+  end
+
+  def create_subscription_aspect(host, system)
+    logger.info("Creating subscription aspect for host #{host.name}.")
+    subscription_aspect = host.subscription_aspect = Katello::Host::SubscriptionAspect.new
+    subscription_aspect.activation_keys = system.activation_keys
+    subscription_aspect.uuid = system.uuid
+
+    if system.backend_data
+      subscription_aspect.service_level = system.backend_data['serviceLevel']
+      subscription_aspect.release_version = system.backend_data['releaseVer']['releaseVer']
```

Comment:
You may want to handle if system.backend_data['releaseVer'] is nil.  Normally i wouldn't worry, but as this is a migration i'd rather be extra careful